src/project-2.js

```diff
@@ -6,6 +6,14 @@ const getBiggest = (x, y) => {
 };
 
 const greeting = (language) => {
+  if (language === 'German') {
```

Comment:
If your base case matches one of your cases, you can just drop the else if (language === 'English').
I'd also suggest using switch cases to reduce code size.